pwndbg/gdblib/heap/__init__.py

```diff
@@ -124,3 +124,48 @@ def resolve_heap(is_first_run: bool = False) -> None:
             )
     else:
         current = pwndbg.gdblib.heap.ptmalloc.DebugSymsHeap()
+
+
+def heap_freebin_address_lookup(addr: int) -> str | None:
```

Comment:
Since this is GLIBC malloc-specific, perhaps it could reside in "pwndbg/gdblib/heap/ptmalloc.py"